base/src/org/spin/model/MADContextInfo.java

```diff
@@ -247,6 +250,7 @@ public static MADContextInfo getFromWindowId(Properties ctx, int windowId) {
 							&& !(value instanceof Double)
 							&& !(value instanceof Float)
 							&& !(value instanceof Integer)
+							&& !(value instanceof Long)
```

Comment:
Please validate it with Type of DB

Reply:
has already been validated with DB data type